bolt-modules/dir/lib/puppet/functions/dir/children.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Returns an array containing all of the filenames except for "." and ".." in the given directory.
+Puppet::Functions.create_function(:'dir::children', Puppet::Functions::InternalFunction) do
+  # @param dirname Absolute path or Puppet module name.
```

Comment:
This function does accept a relative path, which will expand relative to the current working directory.
Would it make sense to have it expand relative to the project instead? I don't really feel strongly either way.

Reply:
I think that makes sense since this is a plan function, and expanding relative to projects makes it more predictable. I'm almost inclined to just fail if users try to use a relative path, but it'd be useful even for things like the faux-external-facts plan where you could just load `facts.d`.